server/src/main/java/io/littlehorse/server/streams/lhinternalscan/publicrequests/SearchWorkflowEventRequestModel.java

```diff
@@ -0,0 +1,117 @@
+package io.littlehorse.server.streams.lhinternalscan.publicrequests;
+
+import com.google.protobuf.Message;
+import io.grpc.Status;
+import io.littlehorse.common.LHStore;
+import io.littlehorse.common.exceptions.LHApiException;
+import io.littlehorse.common.model.getable.objectId.WorkflowEventDefIdModel;
+import io.littlehorse.common.model.getable.objectId.WorkflowEventIdModel;
+import io.littlehorse.common.proto.BookmarkPb;
+import io.littlehorse.common.proto.GetableClassEnum;
+import io.littlehorse.common.proto.TagStorageType;
+import io.littlehorse.common.util.LHUtil;
+import io.littlehorse.sdk.common.proto.SearchWorkflowEventRequest;
+import io.littlehorse.sdk.common.proto.WorkflowEventId;
+import io.littlehorse.sdk.common.proto.WorkflowEventIdList;
+import io.littlehorse.server.streams.lhinternalscan.PublicScanRequest;
+import io.littlehorse.server.streams.lhinternalscan.SearchScanBoundaryStrategy;
+import io.littlehorse.server.streams.lhinternalscan.TagScanBoundaryStrategy;
+import io.littlehorse.server.streams.lhinternalscan.publicsearchreplies.SearchWorkflowEventReply;
+import io.littlehorse.server.streams.storeinternals.index.Attribute;
+import io.littlehorse.server.streams.topology.core.ExecutionContext;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+public class SearchWorkflowEventRequestModel
+        extends PublicScanRequest<
+                SearchWorkflowEventRequest,
+                WorkflowEventIdList,
+                WorkflowEventId,
+                WorkflowEventIdModel,
+                SearchWorkflowEventReply> {
+
+    private Date earliestStart;
+    private Date latestStart;
+    private WorkflowEventDefIdModel workflowEventDefId;
+    private Boolean isClaimed = null;
```

Comment:
What does `isClaimed` mean in this context? It seems like copy-n-paste from `SearchExternalEvent`

Reply:
Good catch, that should have been removed.